docs/development.rst

```diff
@@ -23,6 +23,26 @@ Generate minified JS and CSS files via ``Gulp``:
 Run the test suite with ``tox``. More information about testing is
 available at :doc:`Testing page <testing>`.
 
+Local testing
+-------------
+
+You can test this extension from the docs folder:
+
+.. prompt:: bash
+
+   cd docs
+   pip install sphinx-autobuild
+   pip install -r requirement.txt
```

Comment:
Isn't it the name of this file **requirements.txt** (plural)?

Reply:
Good catch @nienn!